VideoLocker/res/values/text_styles.xml

```diff
@@ -103,19 +103,27 @@
     <style name="bold_white_text" parent="@style/bold_text">
         <item name="android:textColor">@android:color/white</item>
     </style>
+
+    <style name="regular_black_text" parent="@style/regular_text">
+        <item name="android:textColor">@color/black</item>
+    </style>
     
     <style name="regular_grey_text" parent="@style/regular_text">
         <item name="android:textColor">@color/grey_text_mycourse</item>
     </style>
-
-    <style name="light_grey_text" parent="@style/regular_text">
-        <item name="android:textColor">@color/grey_video_size_text</item>
-    </style>
     
     <style name="regular_white_text" parent="@style/regular_text">
         <item name="android:textColor">@android:color/white</item>
     </style>
 
+    <style name="regular_edx_gray_xdark_text" parent="@style/regular_text">
+        <item name="android:textColor">@color/edx_grayscale_neutral_x_dark</item>
+    </style>
+
+    <style name="regular_edx_gray_base_text" parent="@style/regular_text">
+        <item name="android:textColor">@color/edx_grayscale_neutral_base</item>
+    </style>
+

```

Comment:
This should be beneath the `regular_grey_text` style I guess, so that the `regular_white_text` style remains separate.